public/app/features/trails/MetricSelectScene.tsx

```diff
@@ -298,11 +298,11 @@ export class MetricSelectScene extends SceneObjectBase<MetricSelectSceneState> {
       <div className={styles.container}>
         {showHeading && (
```

Comment:
One consequence of having the Text be part of the {showHeading} block is that it doesn't show up in the "Related Metrics" variant of this view.

![image](https://github.com/grafana/grafana/assets/38694490/eaeb05f5-b89f-47e0-9a49-c6bf477fc7a7)